### DISCUSSION TITLE
How do you make mini.ai avoid jumping if it doesn't find a text object?

### DISCUSSION THREAD

**Discussion by lateef-k** [nvim-mini/mini.nvim#160] [Q&A] [answered]
It does this weird thing where it'll jump to the end of the file and show you a little warning if it doesn't find a custom text object, and worse is you can't `ctrl-o` back to position before you jumped. Is there a way to just show a warning and not jump if an object is not found?

**Comment by echasnovski** [comment, marked as answer]
It shouldn't jump if an object is not found. Could you give an example of config, text, and key presses that cause you this behavior?

**Reply by lateef-k** [reply]
I found out I was doing a wrong sequence `[count]v(textobj)` rather than `v[count][textobj]`, it's working fine now. And I can't even seem to reproduce the weird behavior with the former sequence. Thanks for the quick reply though. Should I delete this or just keep it?

**Reply by echasnovski** [reply]
> I found out I was doing a wrong sequence `[count]v(textobj)` rather than `v[count][textobj]`, it's working fine now. And I can't even seem to reproduce the weird behavior with the former sequence. Thanks for the quick reply though. Should I delete this or just keep it?

Glad to hear that! Enjoy 'mini.ai' :)

Let's not delete this.
